code/game/turfs/simulated/wall_icon.dm

```diff
@@ -1,6 +1,24 @@
 /**
  * generates damage overlays
  */
+
+GLOBAL_REAL_VAR(neighbor_typecache) = typecacheof(list(
+	/obj/machinery/door/airlock,
+	/obj/machinery/door/firedoor,
+	/obj/machinery/door/blast,
+	/obj/structure/filler_object,
+	/obj/structure/window/reinforced/tinted/full,
+	/obj/structure/window/reinforced/full,
+	/obj/structure/window/phoronreinforced/full,
+	/obj/structure/window/phoronbasic/full,
+	/obj/structure/window/basic/full,
+	/obj/structure/window/reinforced/polarized/full,
+	/obj/structure/wall_frame,
+	/obj/machinery/smartfridge,
+	))
+
+GLOBAL_REAL_VAR(wall_overlays_cache) = list()
```

Comment:
make this and other overlays caches managed, please
the very slight speedup from this being a real global doesn't outweigh being unable to VV this and others like it